girder/utility/install.py

```diff
@@ -104,6 +105,8 @@ def _pipeOutputToProgress(proc, progress):
                 buf = os.read(pipe.fileno(), 1024)
                 if buf:
                     buf = buf.decode('utf8', errors='ignore')
+                    # Remove ANSI escape sequences
+                    buf = re.sub('(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]', '', buf)
```

Comment:
There is a discussion here: http://stackoverflow.com/questions/14693701/how-can-i-remove-the-ansi-escape-sequences-from-a-string-in-python and here: https://en.wikipedia.org/wiki/ANSI_escape_code .

It shouldn't matter whether this is done on the bytes or the unicode string, as all of the characters involved get passed through without modification.  It would be mildly more efficient to operate on the bytes, since any length reduction would reduce the decoding cost.  I can change it, if you like.